mongo_query.py

```diff
@@ -0,0 +1,49 @@
+from bson.objectid import ObjectId
+from datetime import datetime
+from item import Item
+
+date_time_format = "%Y-%m-%dT%H:%M:%S.%fZ"
+
+def get_items(collection):
+    items = []
+    for item in collection.find():
+        items.append(
+            Item(
+                item['_id'],
+                item['status'],
+                item['name'],
+                item['last_modified']
+            )
+        )
+    return items
+
+def add_new_item(collection, name):
+    collection.insert_one(
+        {
+            "name": name,
+            "status": 'To Do',
+            "last_modified": datetime.now().strftime(date_time_format)
+        }
+    )
+
+def mark_item_as_complete(collection, id):
+    collection.update_one(
+        {"_id": ObjectId(id)},
+        {
+            "$set": {
+                "status": 'Done',
+                "last_modified":  datetime.now().strftime(date_time_format)
+            }
+        }
+    )
+
+def mark_item_as_uncomplete(collection, id):
+    collection.update_one(
+        {"_id": ObjectId(id)},
+        {
+            "$set": {
+                "status": 'To Do',
+                "last_modified": datetime.now()
```

Comment:
Your other functions set this date using `datetime.now().strftime(date_time_format)` - this is slightly different so could lead to inconsistencies. Pulling these out into a function (e.g. `get_current_datetime_for_mongo`) could make it simpler to catch similar mistakes in future